src/app/donation-start/donation-start-form/donation-start-form.component.ts

```diff
@@ -772,9 +772,14 @@ export class DonationStartFormComponent implements AfterContentChecked, AfterCon
       throw new Error("Missing stripe elements");
     }
 
-    const paymentMethod = this.selectedSavedMethod || (
-      await this.stripeService.confirmPaymentWithPaymentElement(this.donation, <StripeElements>this.stripeElements)
-    ).paymentMethod;
+    let paymentMethodResult;
+    let paymentMethod;
```

Comment:
when I hover these the type shows up as `any` - do you know why that doesn't fail the build since we have `"noImplicitAny": true,` in `tsconfig.app.json` ? Maybe it's OK for it to be `any` here because better types are inferred as soon as the assignments are done and that's very close.

Reply:
Mm, yeah good question – not sure I'm afraid